docs/_integration-with-exisiting-apps-objc.md

```diff
@@ -213,24 +213,22 @@ import {
   View
 } from 'react-native';
 
-class RNHighScores extends React.Component {
-  render() {
-    var contents = this.props['scores'].map((score) => (
-      <Text key={score.name}>
-        {score.name}:{score.value}
-        {'\n'}
+const RNHighScores = (props) => {
+  const contents = props['scores'].map((score) => (
```

Comment:
```suggestion
const RNHighScores = ({ scores }) => {
  const contents = scores.map((score) => (
```

Since you are making a refactor, it would be nice to see `props` as destructed object or at least accessed via `.` (dot). The currently used notation is not often used in other examples 🙂

Reply:
hi @Simek  thanks for the refactor suggestions this PR it's ready to be merged thank you :)